string category
float32 obj_score
float32 x 
float32 y
